core/src/main/java/org/microshed/testing/jwt/JwtConfigExtension.java

```diff
@@ -0,0 +1,99 @@
+package org.microshed.testing.jwt;
+
+import org.junit.jupiter.api.extension.AfterEachCallback;
+import org.junit.jupiter.api.extension.BeforeEachCallback;
+import org.junit.jupiter.api.extension.ExtensionConfigurationException;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.microshed.testing.internal.InternalLogger;
+import org.microshed.testing.jupiter.MicroShedTestExtension;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+
+public class JwtConfigExtension implements BeforeEachCallback, AfterEachCallback {
+
+    private static final InternalLogger LOG = InternalLogger.get(JwtConfigExtension.class);
+
+    @Override
+    public void beforeEach(ExtensionContext context) throws Exception {
+        configureJwt(context);
+    }
+
+    @Override
+    public void afterEach(ExtensionContext context) throws Exception {
+        removeJwt(context);
+    }
+
+    private void configureJwt(ExtensionContext context) throws Exception {
```

Comment:
Have a more restrictive throws here as to avoid uncaught exceptions. 
```suggestion
    private void configureJwt(ExtensionContext context) throws ExtensionConfigurationException {
```

Reply:
Instead of:     private void configureJwt(ExtensionContext context) throws ClassNotFoundException, InstantiationException, IllegalAccessException, NoSuchFieldException, NoSuchMethodException, InvocationTargetException {

I've used Exception!

Reply:
We should limit the exception thrown by this method to only `ExtensionConfigurationException`.
All other exceptions should be caught and a meaning message put into the `ExtensionConfigurationException` with the original exception set as `caused by`

Reply:
done!